node/src/components/linear_chain.rs

```diff
@@ -132,22 +201,28 @@ where
         match event {
             Event::Request(LinearChainRequest::BlockRequest(block_hash, sender)) => effect_builder
                 .get_block_from_storage(block_hash)
-                .event(move |maybe_block| Event::GetBlockResult(block_hash, maybe_block.map(Box::new), sender)),
+                .event(move |maybe_block| {
+                    Event::GetBlockResult(block_hash, maybe_block.map(Box::new), sender)
+                }),
             Event::Request(LinearChainRequest::BlockAtHeightLocal(height, responder)) => {
                 effect_builder
                     .get_block_at_height(height)
-                    .event(move |block| Event::GetBlockByHeightResultLocal(height, block.map(Box::new), responder))
+                    .event(move |block| {
+                        Event::GetBlockByHeightResultLocal(height, block.map(Box::new), responder)
+                    })
             }
             Event::Request(LinearChainRequest::BlockAtHeight(height, sender)) => {
                 // Treat `linear_chain` as a cache of least-recently asked for blocks.
                 // match self.linear_chain.get(height as usize).cloned() {
                 //     Some(block) => effect_builder
                 //         .immediately()
-                //         .event(move |_| Event::GetBlockByHeightResult(height, Some(block), sender)),
-                //     None =>
+                //         .event(move |_| Event::GetBlockByHeightResult(height, Some(block),
+                // sender)),     None =>
```

Comment:
Should we remove this comment, and turn it into a ticket?

Reply:
Yeah, I think it's a high time.